### DISCUSSION TITLE
Build official docker image ofr docker hub

### DISCUSSION THREAD

**Discussion by uhlhosting** [teamhanko/hanko#1236] [Ideas]
Any plans to build and push to docker hub an official image people could use to deploy via docker without the need to always build the whole app before deploying.

**Comment by globalexport** [comment]
You could use your own registry to push images to (with Gitlab for instance).

**Comment by FlxMgdnz** [comment]
Sorry, I missed your question @uhlhosting. Github also provides containers, that's what we use:  https://github.com/teamhanko/hanko/pkgs/container/hanko
